custom_components/tahoma/climate_st.py

```diff
@@ -85,9 +87,12 @@ def __init__(self, device_url: str, coordinator: TahomaDataUpdateCoordinator):
         super().__init__(device_url, coordinator)
         self._temp_sensor_entity_id = None
         if self.hvac_mode == HVAC_MODE_AUTO:
-            self._saved_target_temp = self.select_state(
-                PRESET_TEMPERATURES[self.preset_mode]
-            )
+            if self.preset_mode == PRESET_NONE:
+                self._saved_target_temp = None
+            else:
+                self._saved_target_temp = self.select_state(
+                    PRESET_TEMPERATURES[self.preset_mode]
+                )
```

Comment:
Can you please what it means graphically?

Reply:
If target is None, the target temp just doesn't show on the card, and the slider stays at the bottom left end.